apps/src/feedbackBlocks.js

```diff
@@ -1,5 +1,5 @@
 var constants = require('./constants');
-var readonly = require('./templates/readonly.html');
+var readonly = require('./templates/readonly.html.ejs');
```

Comment:
If we want to limit how much change there is, I bet there's a way to tell gulp browserify to look for ejs files when requiring. That said, in the long run, I think I might actually prefer explicitly calling out that these are ejs files.